packages/dashboard/src/messages.ts

```diff
@@ -55,8 +55,40 @@ export type PropertySectionMessages = {
   propertyComponent: PropertyComponentMessages;
 };
 
+export type ChartSettingSectionMessages = {
+  yAxis: {
+    label: string;
+    viewOnChart: string;
+    title: string;
+  };
+
+  legend: {
+    width: string;
+    position: string;
+    showDataColor: string;
+  };
+};
+
 export type SidePanelMessages = {
   propertySection: PropertySectionMessages;
+  chartSettingSection: ChartSettingSectionMessages;
+  baseSettings: {
+    width: string;
+    height: string;
+    x: string;
+    y: string;
+    title: string;
+  };
+
+  textSettings: {
+    title: string;
+    font: string;
+    color: string;
+    style: string;
+    size: string;
+    horiz: string;
```

Comment:
What is horiz used for? should it be horizontal?

Reply:
It is for horizontal alignment of the text. Update the naming.